frontend/javascripts/oxalis/view/settings/histogram_view.js

```diff
@@ -138,8 +138,10 @@ class Histogram extends React.PureComponent<HistogramProps> {
     }
     ctx.stroke();
     ctx.closePath();
-    activeRegion.lineTo(((intensityRangeMax - minRange) / fullLength) * canvasWidth, 0);
-    activeRegion.lineTo(((intensityRangeMin - minRange) / fullLength) * canvasWidth, 0);
+    const activeRegionRightLimit = Math.min(histogramMax, intensityRangeMax);
+    const activeRegionLeftLimit = Math.max(histogramMin, intensityRangeMin);
+    activeRegion.lineTo(((activeRegionRightLimit - minRange) / fullLength) * canvasWidth, 0);
+    activeRegion.lineTo(((activeRegionLeftLimit - minRange) / fullLength) * canvasWidth, 0);
```

Comment:
@MichaelBuessemeyer My assumption would be that `histogramMax` is always <= `intensityRangeMax` and therefore, it should be sufficient to simply use `histogramMax` in the `activeRegion.lineTo` call. Is that assumption correct?

Reply:
Maybe I was confusing the `histogramMax` and `intensityRangeMax`. Hmm, could you distinguish what each of those is again for me? Maybe we can find more descriptive names that make the difference more clear.

Reply:
I think you really confused both of them:
- `histogramMax` is the maximum color value that is included in the histogram data send by the backend.
- `intensityRangeMax` is the maximum color value of the displayed color range and is equal to the higher value of the slide. What's new now is that this value can be greater than `histogramMax` as you can see it in your example:
![max_range_float](https://user-images.githubusercontent.com/1702075/84762928-b8b03900-afcb-11ea-84b5-4e34550257e5.png)

To be honest I think the names are pretty good as the `intensityRange` is the range given by the slider and thus `intensityRangeMax` as the upper limit of that range is not far-fetched (in my opinion) -> Feel free to argue about that.

The same goes for `histogramMax` as this is the highest color value included in the histogram data.

Reply:
Ah understood, that makes sense, sorry for the confusion :)